src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final int POSITION_OF_CATEGORY_IN_DATA = 0;
+    private static final int POSITION_OF_COST_IN_DATA = 1;
+
+    private void writeToFile(String toFileName, String data) {
+        try (FileWriter writer = new FileWriter(toFileName)) {
+            writer.write(data);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't open a file" + e);
+        }
+    }
+
+    private String createReport(String fromFileName) {
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            int supplyTotal = 0;
+            int buyTotal = 0;
+            String lineFromFile = bufferedReader.readLine();
 
+            while (lineFromFile != null) {
+                String[] splitResult = lineFromFile.split(",");
```

Comment:
Do I need to take this variable declaration out from the cycle and initialyse it in the cycle?